actors/builtin/miner/monies_test.go

```diff
@@ -73,4 +73,90 @@ func TestPledgePenaltyForTermination(t *testing.T) {
 				bigInitialPledgeFactor))
 		assert.Equal(t, expectedFee, fee)
 	})
+
+	t.Run("fee for replacement = fee for original sector when power, BR are unchanged", func(t *testing.T) {
+		// initialPledge equal to undeclaredPenalty guarantees expected reward is greater
+		initialPledge := undeclaredPenalty
+		dayReward := big.Div(initialPledge, bigInitialPledgeFactor)
+		twentyDayReward := big.Mul(dayReward, bigInitialPledgeFactor)
+		sectorAgeInDays := int64(20)
+		sectorAge := abi.ChainEpoch(sectorAgeInDays * builtin.EpochsInDay)
+		replacementAge := abi.ChainEpoch(2 * builtin.EpochsInDay)
+
+		// use low power, so we don't test SP=SP
+		power := big.NewInt(1)
+
+		// fee for old sector if had terminated when it was replaced
+		unreplacedFee := miner.PledgePenaltyForTermination(dayReward, sectorAge, twentyDayReward, powerEstimate, power, rewardEstimate, big.Zero(), 0)
+
+		// actual fee including replacement parameters
+		actualFee := miner.PledgePenaltyForTermination(dayReward, replacementAge, twentyDayReward, powerEstimate, power, rewardEstimate, dayReward, sectorAge-replacementAge)
+
+		assert.Equal(t, unreplacedFee, actualFee)
+	})
+
+	t.Run("fee for replacement = fee for same sector without replacement after 70 days", func(t *testing.T) {
+		// initialPledge equal to undeclaredPenalty guarantees expected reward is greater
+		initialPledge := undeclaredPenalty
+		dayReward := big.Div(initialPledge, bigInitialPledgeFactor)
+		twentyDayReward := big.Mul(dayReward, bigInitialPledgeFactor)
+		sectorAgeInDays := int64(20)
+		sectorAge := abi.ChainEpoch(sectorAgeInDays * builtin.EpochsInDay)
+		replacementAge := abi.ChainEpoch(71 * builtin.EpochsInDay)
+
+		// use low power, so we don't test SP=SP
+		power := big.NewInt(1)
+
+		// fee for new sector with not replacement
+		noReplace := miner.PledgePenaltyForTermination(dayReward, replacementAge, twentyDayReward, powerEstimate, power, rewardEstimate, big.Zero(), 0)
+
+		// actual fee including replacement parameters
+		withReplace := miner.PledgePenaltyForTermination(dayReward, replacementAge, twentyDayReward, powerEstimate, power, rewardEstimate, dayReward, sectorAge)
+
+		assert.Equal(t, noReplace, withReplace)
+	})
+
+	t.Run("fee for replacement = fee for same sector without replacement after 70 days", func(t *testing.T) {
```

Comment:
This appears to be a copy-paste of the test above. Did you mean to change this to exercise the <70d case?

Reply:
The test below tests the <70 day case. Looks like this just got duplicated somehow.